quattor/client/rpms.pan

```diff
@@ -29,6 +28,11 @@ include { 'rpms/package_default_versions' };
     if(! exists(spma)) {
         SELF[spma] = nlist();
     };
+    ncd = escape('ncm-ncd');
```

Comment:
`multi` doesn't work anymore, IIRC. In any case, `pkg_add` or `pkg_repl` should be the way to go here, indeed, @kwaegema, can you re-check?